packages/edge-bundler/node/npm_dependencies.ts

```diff
@@ -70,9 +73,22 @@ const getTypesPath = async (packageJsonPath: string): Promise<string | undefined
   return await getTypePathFromTypesPackage(name, packageJsonPath)
 }
 
-const safelyDetectTypes = async (packageJsonPath: string): Promise<string | undefined> => {
+function packageName(specifier: string) {
+  if (!specifier.startsWith('@')) return specifier.split('/')[0]
+  const [scope, pkg] = specifier.split('/')
+  return scope + '/' + pkg
+}
+
+const safelyDetectTypes = async (pkg: string, basePath: string): Promise<string | undefined> => {
+  const lookupPath = basePath.endsWith(path.sep) ? basePath : basePath + path.sep
```

Comment:
Do we need this? Can we not just pass `basePath` to `cwd` in `findUp`?

Reply:
you are right, that works 👍